pkg/test/framework/components/istio/eastwest.go

```diff
@@ -81,6 +81,7 @@ func (i *operatorComponent) deployEastWestGateway(cluster cluster.Cluster, revis
 		"--set", "tag=" + imgSettings.Tag,
 		"--set", "values.global.imagePullPolicy=" + imgSettings.PullPolicy,
 		"--set", "values.gateways.istio-ingressgateway.autoscaleEnabled=false",
+		"--set", "values.gateways.istio-ingressgateway.injectionTemplate=gateway",
```

Comment:
We should test what we ship to customers. I am 100% fine with this change, but it should be in `genGatewayScript`, not the test code

Reply:
Updated